### DISCUSSION TITLE
Data import/export feature

### DISCUSSION THREAD

**Discussion by ghost** [payloadcms/payload#2425] [Feature Requests & Ideas]
Hi, thanks for creating this amazing product.

After creating the collection, I'm looking for a way to import data from CSV, JSON files. I think it will be a useful feature if users can import data into cms.

**Comment by matthijs166** [comment]
This should be a plugin, I think, and not a default payload feature

**Comment by leikoilja** [comment]
Looking for the same functionality, especially for the https://github.com/payloadcms/plugin-form-builder/issues/47

Edit: as a temp manual work-around, since payload csm is using mongo db I use MongoDB Compass app, connect to the db and since it's a document storage - i can simply export it into json or csv. Very manual, but better than manual copy-paste 😅

**Reply by webmuch** [reply]
You can make your own:

https://github.com/payloadcms/plugin-form-builder/issues/47#issuecomment-1707296877
